File: test/fixtures/output/r/httr/jsonObj-multiline.r
library(httr)

url <- "http://mockbin.com/har"

payload <- "{\n  \"foo\": \"bar\"\n}"

encode <- "json"

response <- VERB("POST", url, body = payload, add_headers(content_type = 'application/json'), encode = encode)

content(response, "text")

File: src/targets/node/unirest/fixtures/jsonObj-multiline.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'application/json'
});

req.type('json');
req.send({
  foo: 'bar'
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});